enterprise/frontend/src/embedding-sdk/components/private/QuestionEditor/QuestionEditor.tsx

```diff
@@ -137,19 +137,21 @@ const QuestionEditorInner = () => {
 
 export const QuestionEditor = ({
   questionId,
-  isSaveEnabled = true,
   onBeforeSave,
   onSave,
   plugins,
   entityTypeFilter,
+  isSaveEnabled,
```

Comment:
is the removal of `isSaveEnabled` default value here still valid?

Reply:
@WiNloSt Good catch, I indeed missed that. Fixed in https://github.com/metabase/metabase/pull/49251/commits/79b87dcd2c873a06a91248310c17902b5e5c4893